plugins/polio/api/vaccines/supply_chain.py

```diff
@@ -656,7 +726,12 @@ class VaccineRequestFormViewSet(ModelViewSet):
     # feature flag
     # - add GET arrival reports and pre alerts
 
-    permission_classes = [VaccineSupplyChainReadWritePerm]
+    permission_classes = [
+        lambda: VaccineStockManagementPermission(
+            non_admin_perm=permission.POLIO_VACCINE_SUPPLY_CHAIN_READ,
+            admin_perm=permission.POLIO_VACCINE_SUPPLY_CHAIN_WRITE,
```

Comment:
It's confusing that you put them in "non-admin, admin" order where in the constructor you did "admin, non-admin."

This should be consistent